src/states_screens/tracks_screen.cpp

```diff
@@ -142,7 +142,7 @@ void TracksScreen::beforeAddingWidget()
     if (group_amount > 1)
     {
         //I18N: name of the tab that will show tracks from all groups
-        tabs->addTextChild( _("All"), ALL_TRACK_GROUPS_ID );
+        tabs->addTextChild( _(ALL_TRACK_GROUPS_ID), ALL_TRACK_GROUPS_ID );
```

Comment:
This may seem like nitpicking, but the label used to be "All" with a capital A, whereas ALL_TRACK_GROUPS_ID is "all" in lowercase. Furthermore, ALL_TRACK_GROUPS_ID is meant to be an ID, not a label; because the ID is similar to the label is IMO not a good reason to confuse IDs and labels, and I think we should always clearly separate ID and label. Translating an ID and then using it as label seems all wrong